homework/index.js

```diff
@@ -30,14 +30,77 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    leftContainer.innerHTML = '';
+    const table = createAndAppend('table', leftContainer);
+    const tBody = createAndAppend('tBody', table);
+    const trRepository = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trRepository, { text: 'Repository: ' });
+    createAndAppend('a', trRepository, {
+      text: repository.name,
+      href: repository.html_url,
+    });
+    const trDescription = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trDescription, { text: 'Description: ' });
+    createAndAppend('td', trDescription, { text: repository.description });
+    const trForks = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trForks, { text: 'Forks: ' });
+    createAndAppend('td', trForks, { text: repository.forks });
+    const trUpdated = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trUpdated, { text: 'Updated: ' });
+    createAndAppend('td', trUpdated, { text: repository.updated_at.toLocaleString() });
```

Comment:
You were in the right direction in trying to format the date in a user-friendly format, but it didn't quite achieve the desired result. The property `repository.updated_at` is just a string and JavaScript cannot possible know it represents a date. The `.toLocaleString()`method you use here has no effect and will not format the string as a date. You first need to convert the string into a date and then apply `.toLocaleString()` (now a method of the `Date` object) to format it:

```js
createAndAppend('td', trUpdated, { text: new Date(repository.updated_at).toLocaleString() });
```

Reply:
Hello @remarcmij code has been improved based on your comment.
Thanks.